librocksdb_sys/crocksdb/c.cc

```diff
@@ -2619,6 +2619,18 @@ void crocksdb_options_set_compression_options(crocksdb_options_t* opt,
   opt->rep.compression_opts.zstd_max_train_bytes = zstd_max_train_bytes;
 }
 
+void crocksdb_options_set_bottommost_level_compression_options(
```

Comment:
```suggestion
void crocksdb_options_set_bottommost_compression_options(
```
to make the naming consistent